src/app/shared/components/notification/notification.component.scss

```diff
@@ -0,0 +1,41 @@
+$animation-duration: 0.3s;
+$animation-function: ease;
+$animation-fill-mode: forwards;
+
+:host {
+  --gradient-degree: 90deg;
+  --status-color-width: 5%;
```

Comment:
Do you have any specific reason to mix this this approach with create css variables?

Reply:
I am not sure what you are referring to when you say "create" CSS variables. Could you elaborate?